README.md

```diff
@@ -1512,6 +1512,52 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+## Schema Generation
+To generate a GraphQL schema, we first map a database schema into an intermediate representation:
+the `SchemaGraph`.
+
+#### SchemaGraph
+* `SchemaGraph`
+    * Represents a database's schema through `SchemaElement` objects.
+        * `SchemaElement`
+            * Can be one of three kinds:
+                * `EdgeType`
+                * `VertexType`
+                * `NonGraphElement`.
+            * Have the following attributes:
+                * `name`
+                * `abstract`
```

Comment:
Could you explain how this attribute comes into play? If this is used for discerning between type/interface VertexTypes, you can add a sentence explaining this is relevant only for VertexType.

Reply:
I forgot to mention, but we also do not include abstract edges in the GraphQL schema. (I fixed this). We also do not represent non-abstract NonGraphElements. 

All of this seems too much to include in this section, so I'll leave these explanations in: "Mapping the SchemaGraph to the GraphQL schema."